lua/octo/utils.lua

```diff
@@ -1570,8 +1570,8 @@ function M.get_lines_from_context(calling_context)
     line_number_start = vim.fn.line "."
     line_number_end = line_number_start
   elseif calling_context == "visual" then
-    line_number_start = vim.fn.line "v"
-    line_number_end = vim.fn.line "."
+    line_number_start = vim.fn.line "'<"
+    line_number_end = vim.fn.line "'>"
```

Comment:
hey @wd60622 @ldelossa , looks like this PR broke non-range selection. 
Because the caller always pass "visual" as the calling_context, when not in the visual mode, this function will always return `0, 0`.

Reply:
When you say "non range selections" what do you mean? Can you give me steps to reproduce and ill take a look

Reply:
I meant in the normal mode. When I add comment without any visual selection, the comment buffer can still pop up. But when I save the comment, it will fail with an error "failed to create thread". 
I dug into this issue and it turned out that the bug is because the line numbers are 0.

Reply:
it's with a keybind

Reply:
Yup I see. Thanks for info. I think youre onto it with maybe using "visual" argument in every scenario.

Reply:
Opened a PR with a proposed fix for this: https://github.com/pwntester/octo.nvim/pull/825